src/pages/builders/materials.tsx

```diff
@@ -7,18 +9,130 @@ import type { Config } from '@/types/config.type'
 
 export type MaterialsProps = Pick<Config, 'title' | 'keywords' | 'description' | 'urls'>
 
+type MaterialLink = {
+  name: string
+  label: string
```

Comment:
If `label` explicitly refers to an icon, please use `icon` property instead that is more explicit